driver-core/src/main/com/mongodb/client/model/IndexOptions.java

```diff
@@ -491,6 +492,32 @@ public IndexOptions wildcardProjection(final Bson wildcardProjection) {
         return this;
     }
 
+    /**
+     * Gets if the index should exist on the target collection but should not be used by the query
```

Comment:
The `should exist` part seems unnecessary. We're creating an index, so it's going to exist...  Consider instead

> Gets whether the index should not be used by the query planner....

Reply:
Updated.